### PR TITLE
added header and footer

### PR DESCRIPTION
**Pull request by Ghostavio**

### CODE REVIEW

File: src/routes.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ export default () => {
   return (
     <Route path="/:lang/:newprojects/" component={App}>
       { /* Home (main) route */ }
-      <IndexRoute component={MapListing}/>
+      <IndexRoute component={App}/>
```
Comment (davidecantoni) [inline src/routes.js]: this doesn't makes sense, please keep MapListing

File: src/containers/App/App.js [hunk 2]
```diff
@@ -61,19 +61,19 @@ export default class App extends Component {
 
   render() {
     const styles = require('./App.scss');
+    console.log(this.props.children);
```
Comment (davidecantoni) [inline src/containers/App/App.js]: Please remove

File: src/containers/App/App.js [hunk 3]
```diff
@@ -61,19 +61,19 @@ export default class App extends Component {
...
     return (
       <div className={styles.app}>
-        <DocumentMeta {...config.app.meta}/>
+        <DocumentMeta {...config.app.meta} />
 
         <Header />
 
-        <div className={styles.appContent}>
```
Comment (davidecantoni) [inline src/containers/App/App.js:70]: Please keep it, it is duplicating because of the routing change that you did.

Comment (Ghostavio) [inline src/containers/App/App.js:70]: oh, ok

File: src/containers/App/App.js (hunk 2 above)
Comment (Ghostavio) [inline src/containers/App/App.js]: removed

File: src/routes.js (hunk 1 above)
Comment (Ghostavio) [inline src/routes.js]: kept :)

**Pull request merged**